### ISSUE TITLE
Propose this list to the aggregators after its completion

### ISSUE THREAD

**Issue by amivanoff** [w3c-cg/awesome-semantic-shapes#6]
Let's track interesting awesome lists aggregators here
 - [sindresorhus/awesome](https://github.com/sindresorhus/awesome) - awesome list of lists
 - [awesomelists.top](https://awesomelists.top/#/) - directory of awesomes
 - [trackawesomelist](https://www.trackawesomelist.com) - [github](https://github.com/trackawesomelist/trackawesomelist/) - daily activity tracker

**Comment by VladimirAlexiev** [comment]
- I think we should insert in https://github.com/sindresorhus/awesome?tab=readme-ov-file#computer-science, right after "Data science"
- @amivanoff do you know how trackawesomelist is collected? Do they use [sindresorhus/awesome](https://github.com/sindresorhus/awesome), or have own submission method, or do their own crawling?

**Comment by VladimirAlexiev** [comment]
@amivanoff We should not delay this. Please first comment on #28

**Comment by amivanoff** [comment]
> I think we should insert in https://github.com/sindresorhus/awesome#computer-science, right after "Data science"

According to the https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md:

"Your entry should be added at the bottom of the appropriate category". So, no after "Data science" for us, just the bottom of the Computer Science, pun intended :)

"Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file."

Our "obsolete software" may be an issue. Due to the "community-based" reviewing process (new submitters should review two other new submissions) it is an easy target for the novice reviewers.

"Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge)."

"All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a Footnotes section at the bottom of the readme. The section should not be present in the Table of Contents."

I'll make a PR for this two.

> @amivanoff do you know how trackawesomelist is collected? Do they use [sindresorhus/awesome](https://github.com/sindresorhus/awesome), or have own submission method, or do their own crawling?

They have their own awesomes list in a Yaml config and their own categories. https://www.trackawesomelist.com/#contribution

**Comment by TallTed** [comment]
> "Your entry should be added at the bottom of the appropriate category".

I strongly disagree with this. It effectively results in a random ordering, which typically makes such lists difficult to consume.

I think the entries within a given category should have a definite ordering, which I suggest be alpha-numeric by name/title. (I don't think there are any categories listing people, but if they should ever exist, I suggest these be ordered by family name.)

**Comment by amivanoff** [comment]
https://awesome.ecosyste.ms -- another indexing service, this time with API
